test/test_scripts/test_session.py

```diff
@@ -1,24 +1,22 @@
 """Unit tests for module /scripts/init/session.py."""
 from datetime import datetime
-from scripts.session import Session
-from scripts import utils
-import time
 import unittest
+from scripts.session import update_session_status
+from scripts import utils
 
 
 class TestSession(unittest.TestCase):
     """Unit tests for class Session."""
 
     @classmethod
-    def setUpClass(self):
+    def setUpClass(cls):
         """Execute this before the tests."""
         pass
 
     @staticmethod
     def get_test_case_name():
         """Generate unique name for unit test case."""
-        time.sleep(1)
-        test_case_name = 'test {}'.format(datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+        test_case_name = 'test ' + datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
```

Comment:
Maybe it would make sense to reuse `get_test_case_name` instead of having it defined in every test file.

Reply:
Moved to utils class